casepro/statistics/migrations/0008_dailyminutetotalcount.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statistics', '0007_populate_label_totals'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DailyMinuteTotalCount',
```

Comment:
@rowanseymour For that many cases a data migration to calculate historic values is going to be slow and database intensive. Are you sure it is worth it for your use-case rather than just having totals on new cases?